tsconfig.json

```diff
@@ -13,9 +13,13 @@
     "sourceMap": true,
     "rootDir": "."
   },
+  "include": [
```

Comment:
I guess that if we use the `include` section, the `exclude` section is no longer necessary.